Elevel/Application/Features/QuestionCommands/UpdateQuestionCommand.cs

```diff
@@ -12,12 +12,7 @@ public class UpdateQuestionCommand
     {
         public class Request : IRequest<Response>
         {
-            public Guid Id { get; set; }
-            public string NameQuestion { get; set; }
-            public DateTimeOffset CreationDate { get; set; }
-            public bool Deleted { get; set; }
-            public Guid AnswerId { get; set; }
-            public Guid? AuditionId { get; set; }
+            public Guid? Id { get; set; }
```

Comment:
Why did you delete all these fields?
What are going to update?

Reply:
Undeleted